rest-api-spec/src/yamlRestTest/resources/rest-api-spec/test/cluster.desired_balance/10_basic.yml

```diff
@@ -19,6 +19,18 @@ setup:
   - gte: { stats.reconciliation_time_in_millis: 0 }
   - match: { routing_table: {} }
 
+---
+"Test cluster_balance_stats":
+
+  - skip:
+      version: " - 8.6.99"
+      reason: "Field added in in 8.7.0"
+
+  - do:
+      _internal.get_desired_balance: { }
+
+  - is_true: 'cluster_balance_stats'
```

Comment:
Hm. This only displays the stats for the tiers with nodes that are actually present in cluster. Do we have a guarantee that `data` or `data_content` nodes are always there or the cluster is randomly generated? Otherwise, the structure and numbers are asserted in corresponding unit test

Reply:
:+1: It is challenging to predict exact values here, but I added a test that verifies overall json structure.

Reply:
Great, thanks.